fuse/opcode.go

```diff
@@ -99,6 +99,9 @@ func doInit(server *Server, req *request) {
 		CongestionThreshold: uint16(server.opts.MaxBackground * 3 / 4),
 		MaxBackground:       uint16(server.opts.MaxBackground),
 	}
+	if server.opts.MaxReadAhead != 0 {
+		out.MaxReadAhead = uint32(server.opts.MaxReadAhead)
```

Comment:
shouldn't this value be capped by the input? Can you find out (maybe look at libfuse source code.)